src/denoising_model.jl

```diff
@@ -0,0 +1,34 @@
+export denoising_model
+
+include("denoising_data.jl")
+
+function denoising_model(shape, shape_p, KERNEL_SIZE, KERNEL_SIGMA = 1.5)
+  sigma = 10^-3
```

Comment:
Does this model only work in Float64? Could `sigma` be made generic?

Reply:
In the literature, no one seemed to be interested by this problem on a lower precision.
I'm also afraid, that the wavelet or Fourier transform might not be generic. I should check.